extensions/wikia/ThemeDesigner/ThemeSettings.class.php

```diff
@@ -11,8 +11,14 @@ class ThemeSettings {
 	const MIN_WIDTH_FOR_SPLIT = 1030;
 	const MIN_WIDTH_FOR_NO_SPLIT = 2000;
 
+	// Keep in sync with $wds-community-header-background-height-small-breakpoint
+	const COMMUNITY_HEADER_BACKGROUND_HEIGHT = 115;
+	// Keep in sync with $wds-community-header-background-width-small-breakpoint
+	const COMMUNITY_HEADER_BACKGROUND_WIDTH = 471;
+
 	const WordmarkImageName = 'Wiki-wordmark.png';
 	const BackgroundImageName = 'Wiki-background';
+	const CommunityHeaderBackgroundImageName = 'Community-header-background';
 	const FaviconImageName = 'Favicon.ico';
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this constant "FaviconImageName" to match the regular expression ^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS115)